specification/compute/resource-manager/Microsoft.Compute/stable/2022-01-03/GalleryRP/gallery.json

```diff
@@ -2149,6 +2159,20 @@
         "remove"
       ]
     },
+    "UserArtifactSettings": {
+      "type": "object",
+      "properties": {
+        "packageFileName": {
+          "type": "string",
+          "description": "Optional. The file to rename the downloaded package to on the VM. If not present, then the file will be called <app name>. This is limited to 4096 characters."
```

Comment:
is it literally going to be <app name> ?
Where does this app name come from?

Reply:
rewrote the comments